src/course-outline/section-card/SectionCard.jsx

```diff
@@ -107,67 +109,86 @@ const SectionCard = ({
     >
       <BaseTitleWithStatusBadge
         title={displayName}
-        status={sectionStatus}
+        status=""
         namePrefix={namePrefix}
       />
     </TitleButton>
   );
 
   return (
-    <div
-      className="section-card"
-      data-testid="section-card"
-      ref={currentRef}
+    <ConditionalSortableElement
+      id={id}
+      draggable={actions.draggable}
+      componentStyle={{
```

Comment:
We can perhaps move all the common styles of these elements to `ConditionalSortableElement.jsx`, so we only have to override items like background here.

Reply:
Good idea. Updated.